.env.example

```diff
@@ -5,4 +5,4 @@ PG_DB=api
 PG_PORT=5432
 PG_HOST=localhost
 PG_USER=amida
-PG_PASSWD=''
\ No newline at end of file
+PG_PASSWD=amida
```

Comment:
NEVER commit passwords to version control. This is a dummy password so here it does not matter, but only make changes in your local `.env`, not the `.example`.